invenio_rdm_records/services/permissions.py

```diff
@@ -214,6 +214,23 @@ class RDMRecordPermissionPolicy(RecordPermissionPolicy):
     can_draft_media_update_files = can_review
     can_draft_media_delete_files = can_review
 
+    #
+    # Media files - record
+    #
+    can_read_media_files = [
+        IfRestricted("record", then_=can_view, else_=can_all),
+        ResourceAccessToken("read"),
+    ]
+    can_get_content_media_files = [
+        # note: even though this is closer to business logic than permissions,
+        # it was simpler and less coupling to implement this as permission check
+        IfFileIsLocal(then_=can_read, else_=[SystemProcess()])
+    ]
+    can_create_media_files = [Disable()]
+    can_set_content_media_files = [Disable()]
+    can_commit_media_files = [Disable()]
+    can_update_media_files = [Disable()]
+    can_delete_media_files = [Disable()]
```

Comment:
```suggestion
    can_media_read_files = [
        IfRestricted("record", then_=can_view, else_=can_all),
        ResourceAccessToken("read"),
    ]
    can_media_get_content_files = [
        # note: even though this is closer to business logic than permissions,
        # it was simpler and less coupling to implement this as permission check
        IfFileIsLocal(then_=can_read, else_=[SystemProcess()])
    ]
    can_media_create__files = [Disable()]
    can_media_set_content_files = [Disable()]
    can_media_commit_files = [Disable()]
    can_media_update_files = [Disable()]
    can_media_delete_files = [Disable()]
```

we are missing

```
permission_action_prefix = "media_"
```
in 
https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/services/config.py#L477

thats why it was working with incorrect names (taking the permissions from the files below, not the newly added with incorrect naming)